src/schema/schema.js

```diff
@@ -28,7 +28,7 @@ schema.aggregate = {
     Q: VALID_AGG_OPS,
     O: ['median','min','max'],
     N: [],
-    T: ['mean', 'median', 'min', 'max'],
+    T: ['mean', 'median', 'min', 'max'], // TODO: revise what should time support
```

Comment:
There are so many aggregate function that Vega now supports ... but not all of them will be supported by time.  
For example, `q1`, `q3` should be supported by time.